plenccodec/wrapper.go

```diff
@@ -153,9 +150,6 @@ func (c WTLengthSliceWrapper) Read(data []byte, ptr unsafe.Pointer, wt plenccore
 			return 0, fmt.Errorf("invalid varint for slice entry %d", i)
 		}
 		offset += n
-		if s == 0 {
```

Comment:
This check was explicitly added as part of a previous fix. Need to check why.

Reply:
Doesn't seem like there was a strong reason. No-one immediately took the fix as far as I can tell